scanner/src/routes/+page.svelte

```diff
@@ -46,8 +52,10 @@
   async function startScan() {
     // Check camera permission
     // This is just a simple example, check out the better checks below
-    await BarcodeScanner.checkPermission({ force: true })
+    const permissionResult = await BarcodeScanner.checkPermission({ force: true })
+    if (!permissionResult.granted) return;
```

Comment:
Please add a TODO here, because we can't just ignore permissions. However I believe we should handle this in a separate task, just want it to be easy to find this later.